### ISSUE TITLE
Wait for other transactions

### ISSUE THREAD

**Issue by bedeho** [Joystream/atlas#6334]
Tried to make a bid on an NFT, I get this, despite not obviously being involved in any transaction

![Screenshot 2024-05-26 at 15 31 33](https://github.com/Joystream/atlas/assets/437292/b9cc0e5a-106a-4952-a702-3a0c17a624c1)

I actually wonder: why do we have this sort of blocking at all? It should be perfectly fine to submit a new transaction at any time, what am I forgetting?

**Comment by ikprk** [comment]
I assume that is because we don't want to allow any user to make an action when the data might change after the prev transaction gets processed. 

I cannot think of any other reason why would we do it. Especially when most of the big transactions display a progress modal that blocks user from making other actions. 

@kdembler do you recall why such thing was implemented?

**Comment by kdembler** [comment]
> I assume that is because we don't want to allow any user to make an action when the data might change after the prev transaction gets processed.

Yeah I think that was it. But definitely there's place for improvements, there's no reason for comment posting/liking to block anything for example.

**Comment by bedeho** [comment]
So you mean like trying to sell or put an NFT on auction that you dont even own yet? Another solution would just be to then minimize the progressing effort into the snackbar, with its own sort of small progress indicator or something like we do in Pioneer. We leave UI as if - like it is before action is taken, and when action is completed, the UI will update to reflect your ownership. What do you think?
